course-exercises/slices/1.go

```diff
@@ -0,0 +1,108 @@
+package slices
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/temaniarpit27/polygon-core-developer-course/helper"
+)
+
+func Exercise1(arr []int, n int) []int {
```

Comment:
Could you give function names according to their's functionality and just left comments about numbers of exercises?

Reply:
done